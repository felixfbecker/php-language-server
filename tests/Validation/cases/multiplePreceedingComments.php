<?php

class Foo
{
    /**
     * Foo
     *
     * @return \Iterator
     */
    // @codingStandardsIgnoreLine
    public function fn()
    {
        return null;
    }
}