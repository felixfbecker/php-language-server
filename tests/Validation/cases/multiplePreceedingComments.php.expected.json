{
    "references": [],
    "definitions": {
        "Foo": {
            "fqn": "Foo",
            "extends": [],
            "isMember": false,
            "roamed": false,
            "isStatic": false,
            "canBeInstantiated": true,
            "symbolInformation": {
                "name": "Foo",
                "kind": 5,
                "location": {
                    "uri": "./multiplePreceedingComments.php"
                },
                "containerName": ""
            },
            "type": null,
            "declarationLine": "class Foo",
            "documentation": null,
            "signatureInformation": null
        },
        "Foo->fn()": {
            "fqn": "Foo->fn()",
            "extends": [],
            "isMember": true,
            "roamed": false,
            "isStatic": false,
            "canBeInstantiated": false,
            "symbolInformation": {
                "name": "fn",
                "kind": 6,
                "location": {
                    "uri": "./multiplePreceedingComments.php"
                },
                "containerName": "Foo"
            },
            "type__tostring": "\\Iterator",
            "type": {},
            "declarationLine": "public function fn()",
            "documentation": "Foo",
            "signatureInformation": {
                "label": "()",
                "documentation": "Foo",
                "parameters": []
            }
        }
    }
}